docs/components/typography/test/index.html

```diff
@@ -0,0 +1,299 @@
+        -        --- 1,9 ----
+        -        + This is an important
+        -        + notice! It should
+        -        + therefore be located at
+        -        + the beginning of this
+        -        + document!
+        -
+        -        ! compress the size of the
+        -        ! changes.
+        -
+        -          It is important to spell
+        -      {% endcode %}
+    </figure>
+</section>
+
+<section><!-- Syntax Highlighting -->
+
+    <figure data-dom-diff="typography-syntax-html">
+      <figcaption>HTML Syntax Highlighting Example</figcaption>
+      {% code 'html' %}
+        <script type="text/javascript" src="path/to/script.js"></script>
+        <link rel="stylesheet" href="path/to/style.css" />
+        <!-- HTML Comment -->
+        <form method="POST" action="submit.html">
+          <fieldset>
+            <legend>Login</legend>
+            <p>
+              <label for="username">Username</label>
+              <input type="text" id="username" name="username" />
+            </p>
+            <p>
+              <label for="password">Password</label>
+              <input type="password" id="password" name="password" />
+            </p>
+          </fieldset>
+          <p>
+            <button type="submit">Log In</button>
+          </p>
+          <p>
+            <img src="path/to/file.png" alt="pretty picture" />
+          </p>
+        </form>
+      {% endcode %}
+    </figure>
+    </section>
+
+
+<section>
+    <!-- Syntax Highlighting -->
+    <figure data-dom-diff="typography-syntax-ini">
+        <figcaption>.ini Syntax Highlighting Example</figcaption>
+        {% code 'ini' %}
+-        ; boilerplate
+-        [package]
+-        name = "some_name"
+-        authors = ["Author"]
+-        description = "This is \
+-        a description"
+-
+-        [[lib]]
+-        name = ${NAME}
+-        default = True
+-        auto = no
+-        counter = 1_000
+-      {% endcode %}
+    </figure>
+</section>
+<section>
+    <!-- Syntax Highlighting -->
+    <figure data-dom-diff="typography-syntax-js">
+        <figcaption>JavaScript Syntax Highlighting Example</figcaption>
+        {% code 'javascript' %}
+-        /**
+-         * @description Multi-line comment
+-         * @returns {String}
+-         */
+-        // Inline Comment
+-        var legacyVariable;
+-        const REGEXP = /h(iy|ell)o*/ig;
+-        const array = [ null, true, false ];
+-        const obj = {
+-          number: 42,
+-          integer: 42.0,
+-          s_string: 'single quotes',
+-          d_string: "double quotes"
+-        };
+-        function greet (name: 'World') {
+-          if (foo !== 'bar') {
+-            return 'nope';
+-          } else if (foo || bar) {
+-            return 'maybe';
+-          } else if (foo && bar) {
+-            return 'wut';
+-          }
+-          return `Hello ${name}!`;
+-        }
+-      {% endcode %}
+    </figure>
+</section>
+<section>
```

Comment:
Remove section. There is no Shadow DOM that needs testing.

Reply:
removed